src/components/sets/PopoverMenu.js

```diff
@@ -90,7 +90,7 @@ function PopoverMenuList(props) {
 
   const defaultPalette = palette
     ? palette.map(colorArrayToString)
-    : PALETTE.map(colorArrayToString);
+    : PALETTE.concat([[255, 255, 255], [128, 128, 128], [0, 0, 0]]).map(colorArrayToString);
```

Comment:
What's going on here? Is this just not part of the palette you linked too?

Reply:
Oh right, I forgot to mention this above. When I had some user observations, one comment was that it was unclear how to color cells as white or black. So I added those options to the color picker here even though I did not want those colors to be in the default palette.